src/main/webapp/view/discussion/discussion-comment.js

```diff
@@ -199,6 +201,30 @@ export default class DiscussionComment extends HTMLElement {
     this.appendChild(typePill);
   }
 
+  /**
```

Comment:
Please see my previous comment on changing the method name. I also think this JavaDoc can be improved because mark as button is not explained. If you rename this to setIsAnsweredButton (or a more descriptive method), then you should explain what the button means, not just that you are adding text and an onclick event.

Reply:
Just going to create a tracking issue for these suggestions, so I can get the PR merged